src/guide.ts

```diff
@@ -34,7 +34,7 @@ export interface FormatMixins {
   format?: string | object;
 
   /**
-   * The format type for labels (`"number"` or `"time"` or a [registered custom format type](https://vega.github.io/vega-lite/usage/compile.html#format-type)).
+   * The format type for labels (`"number"` or `"time"` or a [registered custom format type](https://vega.github.io/vega-lite/usage/config.html#custom-format-type)).
```

Comment:
Fix this above as well: `When used with a [custom `"formatType"`](https://vega.github.io/vega-lite/usage/compile.html#format-type), this value will be passed as `format` alongside `datum.value` to the registered function.`